sqs.cfndsl.rb

```diff
@@ -4,7 +4,10 @@
   tags << { Key: 'Environment', Value: Ref(:EnvironmentName) }
   tags << { Key: 'EnvironmentType', Value: Ref(:EnvironmentType) }
 
+  extra_tags = external_parameters.fetch(:extra_tags, [])
```

Comment:
default should be a hash cause we're looping over an expecting a `|key,value|`

Reply:
copy paste fail :S